dask/tests/test_utils.py

```diff
@@ -24,6 +26,16 @@ def test_textblock(myopen, compression):
         assert ''.join(textblock(fn, k, k, compression)).encode() == b''
 
 
+@pytest.mark.parametrize('myopen,compression',
+                         [(open, None), (GzipFile, 'gzip'), (BZ2File, 'bz2'), (LZMAFile, 'xz')])
```

Comment:
Can you use the `SKIP_XZ` mark in the parametrize here as well, and lose the `if` checks below?